features/step_definitions/membership_application_steps.rb

```diff
@@ -2,7 +2,7 @@
  table.hashes.each do |hash|
    attributes = hash.except('user_email')
    user = User.find_by(email: hash[:user_email])
-    if hash['state'] == 'accepted'
+    if hash['state'] == 'accepted' || hash['state'] == 'rejected'
```

Comment:
@weedySeaDragon this one is not represented in https://github.com/AgileVentures/shf-project/pull/128/files - everything else in this PR is. 
So either add this to the 128 branch or make a separate PR with this change?